lens-data/lens/applicable.rkt

```diff
@@ -3,7 +3,7 @@
 (require reprovide/reprovide)
 (reprovide "main.rkt")
 
-(require (only-in "private/base/base.rkt" use-applicable-lenses!))
```

Comment:
Applicable lenses seems like something that belongs in the `lens-common` package